make check is running check_style.sh on rpmbuild/... contents.

Example:
[ykaul@ykaul lago]$ make check
-------------------------------------------------------------
-------------------------------------------------------------
-~      Running style checks                               --
-------------------------------------------------------------
scripts/check_style.sh
-------------------------------------------------------------
-~      Running static checks                              --
-------------------------------------------------------------
flake8
./rpmbuild/BUILD/lago-0.9.2/lib/lago/virt.py:361:1: E101 indentation contains mixed spaces and tabs
./rpmbuild/BUILD/lago-0.9.2/lib/lago/virt.py:361:1: W191 indentation contains tabs
./rpmbuild/BUILD/lago-0.9.2/lib/lago/virt.py:362:1: W191 indentation contains tabs
./rpmbuild/BUILD/lago-0.9.2/lib/lago/virt.py:362:4: E101 indentation contains mixed spaces and tabs
./rpmbuild/BUILD/lago-0.9.2/lib/lago/virt.py:362:21: E128 continuation line under-indented for visual indent
./rpmbuild/BUILD/lago-0.9.2/lib/lago/virt.py:363:1: W191 indentation contains tabs
./rpmbuild/BUILD/lago-0.9.2/lib/lago/virt.py:364:1: E101 indentation contains mixed spaces and tabs
./rpmbuild/BUILD/lago-0.9.2/lib/lago/virt.py:365:1: E101 indentation contains mixed spaces and tabs
./rpmbuild/BUILD/lago-0.9.2/lib/lago/virt.py:365:1: W191 indentation contains tabs
./rpmbuild/BUILD/lago-0.9.2/lib/lago/virt.py:365:8: E126 continuation line over-indented for hanging indent
./rpmbuild/BUILD/lago-0.9.2/lib/lago/virt.py:366:1: E101 indentation contains mixed spaces and tabs
./rpmbuild/BUILD/lago-0.9.2/lib/lago/virt.py:368:1: E101 indentation contains mixed spaces and tabs
./rpmbuild/BUILD/lago-0.9.2/lib/lago/virt.py:368:1: W191 indentation contains tabs
./rpmbuild/BUILD/lago-0.9.2/lib/lago/virt.py:368:5: E126 continuation line over-indented for hanging indent
./rpmbuild/BUILD/lago-0.9.2/lib/lago/virt.py:369:1: W191 indentation contains tabs
./rpmbuild/BUILD/lago-0.9.2/lib/lago/virt.py:369:2: E101 indentation contains mixed spaces and tabs
./rpmbuild/BUILD/lago-0.9.2/lib/lago/virt.py:370:1: E101 indentation contains mixed spaces and tabs
./rpmbuild/BUILD/lago-0.9.2/build/lib/lago/virt.py:361:1: E101 indentation contains mixed spaces and tabs
./rpmbuild/BUILD/lago-0.9.2/build/lib/lago/virt.py:361:1: W191 indentation contains tabs
./rpmbuild/BUILD/lago-0.9.2/build/lib/lago/virt.py:362:1: W191 indentation contains tabs
./rpmbuild/BUILD/lago-0.9.2/build/lib/lago/virt.py:362:4: E101 indentation contains mixed spaces and tabs
./rpmbuild/BUILD/lago-0.9.2/build/lib/lago/virt.py:362:21: E128 continuation line under-indented for visual indent
./rpmbuild/BUILD/lago-0.9.2/build/lib/lago/virt.py:363:1: W191 indentation contains tabs
./rpmbuild/BUILD/lago-0.9.2/build/lib/lago/virt.py:364:1: E101 indentation contains mixed spaces and tabs
./rpmbuild/BUILD/lago-0.9.2/build/lib/lago/virt.py:365:1: E101 indentation contains mixed spaces and tabs
./rpmbuild/BUILD/lago-0.9.2/build/lib/lago/virt.py:365:1: W191 indentation contains tabs
./rpmbuild/BUILD/lago-0.9.2/build/lib/lago/virt.py:365:8: E126 continuation line over-indented for hanging indent
./rpmbuild/BUILD/lago-0.9.2/build/lib/lago/virt.py:366:1: E101 indentation contains mixed spaces and tabs
./rpmbuild/BUILD/lago-0.9.2/build/lib/lago/virt.py:368:1: E101 indentation contains mixed spaces and tabs
./rpmbuild/BUILD/lago-0.9.2/build/lib/lago/virt.py:368:1: W191 indentation contains tabs
./rpmbuild/BUILD/lago-0.9.2/build/lib/lago/virt.py:368:5: E126 continuation line over-indented for hanging indent
./rpmbuild/BUILD/lago-0.9.2/build/lib/lago/virt.py:369:1: W191 indentation contains tabs
./rpmbuild/BUILD/lago-0.9.2/build/lib/lago/virt.py:369:2: E101 indentation contains mixed spaces and tabs
./rpmbuild/BUILD/lago-0.9.2/build/lib/lago/virt.py:370:1: E101 indentation contains mixed spaces and tabs

--- a/setup.cfg
+++ b/setup.cfg
@@ -1,4 +1,4 @@
 [flake8]
 # ignore line break before operator
 ignore = W503
-exclude = */*venv/*,automation-build
+exclude = */*venv/*,automation-build,rpmbuild,dist

